package.json

```diff
@@ -13,51 +13,50 @@
     "contributors:generate": "all-contributors init"
   },
   "dependencies": {
-    "@angular/animations": "^8.2.6",
```

Comment:
You need to add the --prod here:

    "build:lib": "ng build @ngneat/content-loader",